src/components/Badges/index.js

```diff
@@ -1,18 +1,30 @@
 import VerifiedBadge from '../VerifiedBadge';
 
-function Badges({ tokenId, verified }) {
+function Badges({ content }) {
```

Comment:
@khalatevarun my thoughts:
1. Badges vs Badge, the later is more flexible and reusable, say, if we want to put the Badges in another place, maybe the elements wrap the Badges will change, so it becomes harder for Badges to adapt to that change, the more simple of the component, the better to use.
2. content vs explicit parameter list (color, disabled), the latter is more safe and clear to the user of this component, interface should be a contract with user rather than a blurred object that can be anything in the future.